app/screens/node/Node.tsx

```diff
@@ -459,7 +459,7 @@ const Node = ({ history, location }: Props) => {
         <Address
           key="smesherCoinbase"
           type={AddressType.ACCOUNT}
-          address={coinbase}
+          address={coinbase || ''}
```

Comment:
because of a fast new flow , we're supporting smeshing from different folders from different configs ( right now each network may have its own smeshing options) , during node restart we're updating and smeshing process and till we're waiting for updates from the node, for 1 ms , we can get new update from the node where the coinbase not updated yet. It is just an issue with the update and react renderer. The user won't see an empty address

Reply:
seems after https://github.com/spacemeshos/smapp/pull/1334/files#diff-2f272b881ae8ff902e02cbd366f81c899b63396572563819304c10f4ebd5e769R547 merge It, my changes not needed. At least will be rewritten.